src/systemctl/systemctl.c

```diff
@@ -5860,7 +5860,7 @@ static int show(int argc, char *argv[], void *userdata) {
 
         if (show_mode == SYSTEMCTL_SHOW_HELP && argc <= 1)
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                       "This command expects one or more unit names. Did you mean --help?");
+                                       "'help' is not supported as a command verb. Did you mean --help?");
```

Comment:
hmm, uh? i don't get this? aren't you confusing the user here a bit? I mean `systemctl help systemd-journald.service` is a thing, but with the new msg you suggest otherwise?

Reply:
Oh, OK. I misunderstood this completely. Will rework.